doc/src/pair_grace.rst

```diff
@@ -0,0 +1,191 @@
+.. index:: pair_style grace
```

Comment:
This file needs to be properly integrated into the manual by updating `doc/src/Commands_pair.rst` and `doc/src/pair_style.rst`. Most likely also the list of false positives for the spell checker needs updates as well.